CHANGELOG.md

```diff
@@ -37,6 +37,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v0.38.4] - TBD
+
+### Bug Fixes
+
+* (x/auth) [\#5950](https://github.com/cosmos/cosmos-sdk/pull/5950) Fix `IncrementSequenceDecorator` to use is `IsReCheckTx` instead of `IsCheckTx` to allow account sequence incrementing.
```

Comment:
is this the only entry we need to add?

Reply:
Yes -- see https://github.com/cosmos/cosmos-sdk/issues/6006. There were only two PRs added to this release.